examples/games/alien_cake_addict.rs

```diff
@@ -379,12 +375,12 @@ fn gameover_keyboard(mut state: ResMut<State<GameState>>, keyboard_input: Res<In
 fn display_score(mut commands: Commands, asset_server: Res<AssetServer>, game: Res<Game>) {
     commands
         .spawn(NodeBundle {
-            style: Style {
-                margin: UiRect::all(Val::Auto),
+            spacing: Spacing::AUTO_MARGIN,
```

Comment:
I dislike this pattern in general (this is a sloppy way to center things), but fixing that should be a seperate PR.

Reply:
https://github.com/bevyengine/bevy/issues/6498